File: fsharp-giraffe-server/OpenAPI/src/model/NonExistingStreamError.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.AnyType
open OpenAPI.Model.CodedErrorBase
open OpenAPI.Model.NonExistingStreamErrorAllOf

module NonExistingStreamError = 

  //#region NonExistingStreamError


  type NonExistingStreamError = {
    Result : AnyType;
    Msg : AnyType;
    Code : AnyType;
    Stream : string;
  }
  //#endregion
  

File: dart-jaguar/lib/model/non_existing_stream_error.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


import 'package:openapi/model/coded_error_base.dart';

import 'package:openapi/model/non_existing_stream_error_all_of.dart';

part 'non_existing_stream_error.jser.dart';

class NonExistingStreamError {
  
  @Alias('result', isNullable: true,  )
  final Object result;
  
  @Alias('msg', isNullable: true,  )
  final Object msg;
  
  @Alias('code', isNullable: true,  )
  final Object code;
   /* The name of the stream that could not be found.  */
  @Alias('stream', isNullable: false,  )
  final String stream;
  

  NonExistingStreamError(
      

{
    
     this.result = null,  
     this.msg = null,   this.code = null,  
     this.stream = null 
    
    }
  );

  @override
  String toString() {
    return 'NonExistingStreamError[result=$result, msg=$msg, code=$code, stream=$stream, ]';
  }
}

@GenSerializer(nullableFields: true)
class NonExistingStreamErrorSerializer extends Serializer<NonExistingStreamError> with _$NonExistingStreamErrorSerializer {

}

